pmm/pmm2-upgrade-tests.groovy

```diff
@@ -264,7 +264,7 @@ pipeline {
                         docker exec pmm-server yum clean all
                         docker exec pmm-server yum clean metadata
                     """
-                    setupPMMClient(env.SERVER_IP, CLIENT_VERSION, 'pmm2', 'no', 'release', 'yes', 'compose_setup', params.ADMIN_PASSWORD)
+                    setupPMMClient(env.SERVER_IP, CLIENT_VERSION, 'pmm2', 'no', 'release', 'yes', 'compose_setup', params.ADMIN_PASSWORD, 'no')
```

Comment:
I'm not very sure, but it could be insufficient to fix the function call in one place. We may need to fix calls everywhere, including v3 pipelines. Need to evaluate the assumption.